client/scripts/views/pages/members.ts

```diff
@@ -82,28 +163,46 @@ const MembersPage : m.Component<{}, { membersRequested: boolean, membersLoaded:
       m(Table, [
         m('tr', [
           m('th', 'Member'),
-          m('th.align-right', 'Posts/ Month'),
+          m('th.align-right', 'Posts / Month'),
         ]),
-        vnode.state.membersLoaded.map((info) => {
-          const profile = app.profiles.getProfile(info.chain, info.address);
-          return m('tr', [
-            m('td.members-item-info', [
-              m('a', {
-                href: `/${app.activeId()}/account/${info.address}?base=${info.chain}`,
-                onclick: (e) => {
-                  e.preventDefault();
-                  localStorage[`${app.activeId()}-members-scrollY`] = window.scrollY;
-                  navigateToSubpage(`/account/${info.address}?base=${info.chain}`);
-                }
-              }, [
-                m(User, { user: profile, showRole: true }),
+        vnode.state.profilesLoaded.map((profileInfo) => {
+            return m('tr', [
+              m('td.members-item-info', [
+                m('a', {
```

Comment:
You should take a look at how we do links across our codebase. Sometimes it's just a button or onclick uses mithril's redirect. Just confirm this is the convention that's best to be used in this context

Reply:
I see the navigateToSubpage method used in a lot of onclick redirects across the codebase, so I think this is okay? That part was already written by Jake when I started the bug fix.

Reply:
So, `navigateToSubpage` is a tiny function that does the following:
```
export const navigateToSubpage = (...args) => {
  // prepend community if we are not on a custom domain
  if (!app.isCustomDomain()) {
    args[0] = `/${app.activeId()}${args[0]}`;
  }
  m.route.set.apply(this, args);
};
```

Basically, it normalizes links for custom domains. But if search is redirecting users across communities, then we shouldn't necessarily use this function.

Reply:
Gotcha. It shouldn't be redirecting across communities in this context, this just redirects to the user's account in the given community.